chrome_extension/background/event.js

```diff
@@ -124,7 +116,7 @@ mooltipassEvent.onSaveSettings = function(callback, tab, settings) {
 	mooltipassEvent.onLoadSettings();
 }
 
-mooltipassEvent.onGetStatus = function(callback, tab) {
+mooltipassEvent.onGetStatus = function(callback, senderTab, tab) {
```

Comment:
Not sure that I understand what are you speaking about.
Another extensions can not send messages to our extension. We do not listen for external messages.

I can change implementation and do not send tab via args, but get the active tab in the onGetStatus handler.